package.json

```diff
@@ -25,7 +25,6 @@
     "body-parser": "^1.19.0",
     "cors": "^2.8.5",
     "express": "^4.17.0",
-    "mongodb": "^3.3.3",
```

Comment:
ended up not needing mongodb module to connect to atlas. used mongoose instead.

Reply:
Also this may be why my package.json-lock is showing a bunch of a changes, because I did not manually do anything to it.